src/liballoc/rc.rs

```diff
@@ -569,9 +569,33 @@ impl<T: ?Sized> Rc<T> {
     /// ```
     #[stable(feature = "rc_raw", since = "1.17.0")]
     pub fn into_raw(this: Self) -> *const T {
+        let ptr = Self::as_ptr(&this);
+        mem::forget(this);
+        ptr
+    }
+
+    /// Provides a raw pointer to the data.
+    ///
+    /// Without changing any counts or consuming the value. The pointer is valid for as long there
```

Comment:
The first "sentence" here is not a proper sentence.

Reply:
I meant it as continuation of the caption, but you're right, it's not really smooth transition. Changed.